tests/sav/nitmetrics_args3.res

```diff
@@ -7,11 +7,11 @@ Warning: no source file for `excluded`
   * has_package 1
   * has_readme 1
   * md_lines 12
-  * nb_section 0
-  * md_blocks 14
-  * BlockParagraph 4
-  * BlockUnorderedList 1
-  * BlockListItem 4
-  * SpanCode 4
-  * Image 1
+  * MdDocument 1
+  * MdParagraph 8
```

Comment:
Each list item contains a paragraph with the new parser so 4 `BlockParagraph` + 4 `BlockListItem`